src/main/scala/beam/utils/data/synthpop/ScenarioGenerator.scala

```diff
@@ -607,6 +620,11 @@ object SimpleScenarioGenerator extends StrictLogging {
     CsvHouseholdInfoWriter.write(householdFilePath, households)
     logger.info(s"Wrote households information to $householdFilePath")
 
+    val blocks = generatedData.map(_._3).toVector
+    val blockFilePath = s"$pathToOutput/blocks.csv"
+    CsvBlocksInfoWriter.write(blockFilePath, blocks)
+    logger.info(s"Wrote blocks information to $blockFilePath")
```

Comment:
Should this logger be added to the writer directly?

Reply:
Sure. I'll do this once you go through https://github.com/LBNL-UCB-STI/beam/pull/3451#discussion_r791370805 comment. Thanks.